src/ResourceManager/AzureBatch/documentation/upcoming-breaking-changes.md

```diff
@@ -29,6 +29,15 @@
 
 ## Release 5.0.0 - November 2017
 
+- `PSCloudPool` `ResizeError` was replaced by `ResizeErrors`.
+- `PSCloudTask`, `PSJobManagerTask`, `PSJobPreparationTask`, `PSJobReleaseTask`, and `PSStartTask` no longer have `RunElevated`. It is replaced by `UserIdentity`.
+  For tasks which had `RunElevated = true`, you can use a `UserIdentity` with `elevationLevel: Admin`, similarly for tasks which had `RunElevated = false` you
+  you can use a `UserIdentity` with `elevationLevel: NonAdmin`.
```

Comment:
@matthchr Please use the format specified above for this - please include a sample of a previous cmdlet invocation or property usage as 'old'  and please include code for how to achieve the same efect after the change in 'New'.  If these are all similar, having one example from each group would be ok - we need to use this information to build the migration guide in November.